client/package.json

```diff
@@ -12,51 +12,51 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~11.2.0",
-    "@angular/cdk": "^11.2.0",
-    "@angular/common": "~11.2.0",
-    "@angular/compiler": "~11.2.0",
-    "@angular/core": "~11.2.0",
+    "@angular/animations": "~13.2.1",
+    "@angular/cdk": "^13.2.1",
+    "@angular/common": "~13.2.1",
+    "@angular/compiler": "~13.2.1",
+    "@angular/core": "~13.2.1",
     "@angular/flex-layout": "^11.0.0-beta.33",
```

Comment:
Probably want to update this one too.

(It's a bit annoying, since `@angular/flex-layout` doesn't actually have release versions; they just skip from "beta" to "beta one number higher". 🙃)

Reply:
Moved up to 13.0.0-beta.38.